samila/params.py

```diff
@@ -50,3 +50,10 @@ class Projection(Enum):
     LAMBERT = "lambert"
     MOLLWEIDE = "mollweide"
     RECTILINEAR = "rectilinear"
+
+
+ELEMENTS_LIST = ["{0}*math.cos({1})","{0}*math.sin({1})","{0}*{1}","{0}*abs({1})","{0}*math.ceil({1})","{0}*math.floor({1})"]
```

Comment:
I think it's better if we make it this way:
```

ELEMENTS_LIST = ["{0} * math.cos({1})",
                 "{0} * math.sin({1})",
                 "{0} * {1}",
                 "{0} * abs({1})",
                 "{0} * math.ceil({1})",
                 "{0} * math.floor({1})"]
```
This way we can track further function additions.